mckinseyacademy/mcka-theme
# Drag & Drop List Indentation

- Fixed extra indentation of nested list

Steps to verify changes:
- Add a nested "ul" in  drag & drop html section

- Inner "ul" should have no indentation and outer "ul" should have 20px margin from left

@bradenmacdonald Please get this PR reviewed
